app/models/exporter.rb

```diff
@@ -3,12 +3,8 @@ module Exporter
   class << self
     attr_accessor :time
 
-    def initialize(time=Time.now)
-      @time = time - 1.hour
-    end
-
     def import
-      items = Item.where("created_at: >= @time")
+      items = Item.where('created_at >= :hour_ago', :hour_ago => Time.now - 1.hour)
```

Comment:
Line is too long. [83/80]<br>Use the new Ruby 1.9 hash syntax.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.